core/src/main/scala/com/typesafe/tools/mima/core/Problems.scala

```diff
@@ -45,6 +45,7 @@ sealed abstract class Problem extends ProblemRef {
     case DirectAbstractMethodProblem(ref)                 => s"${ref.methodString} does not have a correspondent in $affectedVersion version"
     case ReversedAbstractMethodProblem(ref)               => s"in $affectedVersion version there is ${ref.methodString}, which does not have a correspondent"
     case UpdateForwarderBodyProblem(ref)                  => s"in $affectedVersion version, classes mixing ${ref.owner.fullName} needs to update body of ${ref.shortMethodString}"
+    case NewMixinForwarderProblem(ref)                    => s"in $affectedVersion version, classes mixing ${ref.owner.fullName} needs to rebuild to use the static mixin forwarder method for ${ref.shortMethodString}"
```

Comment:
If you think it makes sense, the message could mention that super calls to `ref.shortMethodString` need to be recompiled to target the new static mixin forwarder. Probably it gets too complicated and the current version is fine.